ssd1351/src/main/java/com/example/upm/androidthings/driversamples/SpiDisplayActivity.java

```diff
@@ -106,6 +114,7 @@ public void run() {
                 display.setCursor((short)5, (short)70);
                 display.print("IoTDK");
                 display.refresh();
+                updateUI("Intel IoTDK");
```

Comment:
In the Sample, the SPI display even prints "Intel" and then prints "IoTDK". i just concatinated them both in the Text View :). I can just add "IoTDK" instead.